src/hotspot/share/classfile/resolutionErrors.cpp

```diff
@@ -108,22 +108,22 @@ ResolutionErrorEntry::ResolutionErrorEntry(Symbol* error, const char* message,
   Symbol::maybe_increment_refcount(_cause);
 }
 
+ResolutionErrorEntry::ResolutionErrorEntry(const char* nest_host_error):
+        _error(nullptr),
+        _message(nullptr),
+        _cause(nullptr),
+        _cause_msg(nullptr),
+        _nest_host_error(nest_host_error) {
```

Comment:
I see freeze() doesn't copy nest_host_error() to C heap.  But now you have to strdup it here.